solutions/observability/streams/management/advanced.md

```diff
@@ -26,16 +26,16 @@ Features provide useful information for AI processes, such as significant events
 
 ## Index configuration [streams-advanced-index-config]
 
-:::{note}
-Processing and schema changes should typically be done through the Streams interface, and none of these configuration processes are required. This feature mainly exists to help advanced users maintain familiar workflows.
+:::{Important}
+Avoid editing components marked as **managed** or any per-data-stream mappings and settings. Processing and schema changes should typically be done through the Streams interface or API, and none of these configuration processes are required. This feature mainly exists to help advanced users maintain familiar workflows.
 :::
 
-For classic streams, you can manually configure:
+For classic streams, you can access the following components:
 
 - [Index templates](../../../../manage-data/data-store/templates.md#index-templates)
 - [Component templates](../../../../manage-data/data-store/templates.md#component-templates)
 - [Pipelines](../../../../manage-data/ingest/transform-enrich.md)
-- [Data streams](../../../../manage-data/data-store/data-streams.md).
+- [Data streams](../../../../manage-data/data-store/data-streams.md)
 
 For both wired ({applies_to}`stack: preview 9.2`) and classic streams, you can manually configure:
```

Comment:
I know this is outside of the scope of the PR, but I just noticed it - does the wired option need an applies to tag for serverless too? The [docs](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/3679/solutions/observability/streams/streams#wired-streams) for wired streams show that the feature is in tech preview for both deployments.